distribution/lib/Standard/Base/0.0.0-dev/src/Data/Any.enso

```diff
@@ -412,3 +413,14 @@ type Any
              2.to_default_visualization_data
     to_default_visualization_data : Text
     to_default_visualization_data self = self.to_json.to_text
+
+    ## UNSTABLE
+       ADVANCED
+
+       Returns the data requested to render a lazy view of the default visualisation.
+    to_lazy_visualization_data : Text
+    to_lazy_visualization_data self text_window_position text_window_size chunk_size =
+        text = self.to_default_visualization_data
+        min_length_for_laziness = chunk_size * (text_window_size.at 0) * (text_window_size.at 1)
+        if text.length <= min_length_for_laziness then text else
+            get_lazy_visualisation_text_window text text_window_position text_window_size chunk_size
```

Comment:
Could we put these as an extension method in Standard.Visualisation.

The circular dependency this introduces into the library structure is not great. We will also move `to_default_visualisation` there as well soon.

```suggestion
## UNSTABLE
   ADVANCED

   Returns the data requested to render a lazy view of the default visualisation.
Any.to_lazy_visualization_data : Vector Integer -> Vector Integer -> Text
Any.to_lazy_visualization_data self text_window_position text_window_size chunk_size =
    text = self.to_default_visualization_data
    min_length_for_laziness = chunk_size * (text_window_size.first) * (text_window_size.second)
    if text.length <= min_length_for_laziness then text else
        get_lazy_visualisation_text_window text text_window_position text_window_size chunk_size
```

By all means put in `Main.enso` within `Standard.Visualisation` for now. We can organise tidying up within library team.

Reply:
Makes sense. Moving this around.